dotcom-rendering/src/components/SetABTests.importable.tsx

```diff
@@ -99,7 +98,7 @@ export const SetABTests = ({
 		};
 
 		const ab = new AB({
-			mvtId,
+			mvtId: mvtId ?? 0,
```

Comment:
Worth noting that `0` is actually an invalid MVT ID and will therefore not fall into any test. As this might be confusing, I’d suggest something clearly wrong like `-1` or `-Infinity` or even `NaN`

Reply:
Will go for `-1`